src/lipizzaner_master.py

```diff
@@ -49,6 +49,9 @@ def run(self):
             clients = self.cc.settings['general']['distribution']['client_nodes']
         accessible_clients = self._accessible_clients(clients)
 
+        # NOTE: Uncomment this line to run Loss Function Based Diversity
+        # self.cc.settings['general']['distribution']['num_clients'] = len(accessible_clients)
+
```

Comment:
Should we uncomment these lines?

Reply:
Definitely Jamal, now that we are seeing improvements with it!